src/main/java/edu/oregonstate/cs361/battleship/Main.java

```diff
@@ -20,8 +23,49 @@ public static void main(String[] args) {
     }
 
     //This function should return a new model
+
+    /* Modified by aoleson on 1/30/17.
+     * aoleson (1/30/17): This function is called when the page loads. It creates a new model of the game,
+     * randomly places the AI's ships on the board, and  stringifies the model with GSON and sends it back.
+     */
     static String newModel() {
-        return "MODEL";
+
+        //Instantiate model, GSON and random objects
+        BattleshipModel modelObj = new BattleshipModel();
+        Gson gson = new Gson();
+        Random rand = new Random();
+        int grid_max = 9;
+        int grid_min = 0;
+
+        //Proof of concept: Randomly place aircraftCarrier
+        int across = rand.nextInt((5 - grid_min) + 1) + grid_min;
```

Comment:
I'm not going to look at your ship placement code too closely until you finish it in another PR, unless you would like me to :) It looks good at a glance though.